common/motion_utils/src/trajectory/conversion.cpp

```diff
@@ -12,12 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "motion_utils/trajectory/tmp_conversion.hpp"
+#include "motion_utils/trajectory/conversion.hpp"
+
+#include "autoware_auto_planning_msgs/msg/detail/path__struct.hpp"
+#include "autoware_auto_planning_msgs/msg/detail/path_point_with_lane_id__struct.hpp"
+#include "autoware_auto_planning_msgs/msg/detail/path_with_lane_id__struct.hpp"
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
```

Comment:
I have a feeling this header is not needed, but I might be wrong.

Reply:
Indeed. Unnecessary dependencies deleted in the new commit [https://github.com/autowarefoundation/autoware.universe/pull/5909/commits/499cd278476678fb7fc79e4acc7f146c4f422da6](https://github.com/autowarefoundation/autoware.universe/pull/5909/commits/499cd278476678fb7fc79e4acc7f146c4f422da6)

1. [deleted] `#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>`
for `lerpOrientation()`, which was moved to another place.





2. `#include "autoware_auto_planning_msgs/msg/detail/path__struct.hpp"` and 
`#include "autoware_auto_planning_msgs/msg/detail/path_point_with_lane_id__struct.hpp"` and 
`#include "autoware_auto_planning_msgs/msg/detail/path_with_lane_id__struct.hpp"` 
for 
`Path`
`PathPointWithLaneId`
`PathWithLaneId`